test/map.spec.js

```diff
@@ -156,47 +210,86 @@ test('test getSubarrayFrom2dArray', t => {
     [9, 0, 1],
     [5, 6, 7],
   ];
-  t.same(map.getSubarrayFrom2dArray(array, xCenter, yCenter, width, height), expected);
+  t.same(getSubarrayFrom2dArray(array, xCenter, yCenter, width, height), expected);
 
   t.end();
 });
 
-test('test traversableCellsInTile returns correct cells for each tile', t => {
-  let expected = [
+
+test('getTileTraversabilityInCells: returns correctly with traversable tile, CPTL=1', t => {
+  MapRewireAPI.__Rewire__('CPTL', 1);
```

Comment:
whenever you rewire something, remember to __ResetDependency__ at the end of the test so you don't accidentally influence other tests